custom_components/tuya_local/devices/della_airconditioner.yaml

```diff
@@ -0,0 +1,162 @@
+name: Della Air Conditioner
+primary_entity:
+  entity: climate
+  dps:
+    - id: 1
+      name: hvac_mode
+      type: boolean
+      mapping:
+        - dps_val: false
+          value: "off"
+        - dps_val: true
+          constraint: mode
+          conditions:
+            - dps_val: cold
+              value: cool
+            - dps_val: hot
+              value: heat
+            - dps_val: wet
+              value: dry
+            - dps_val: wind
+              value: fan_only
+            - dps_val: auto
+              value: auto
+    - id: 2
+      name: temperature
+      type: integer
+      mapping:
+        - constraint: temperature_unit
+          conditions:
+            - dps_val: C
+              scale: 10
+              step: 10
+              range:
+                min: 160
+                max: 320
+            - dps_val: F
+              scale: 10
+              step: 10
+              range:
+                min: 600
+                max: 900
+    - id: 3
+      name: current_temperature
+      type: integer
+      mapping:
+        - scale: 10
+    - id: 4
+      name: mode
+      type: string
+      hidden: true
+    - id: 5
+      name: fan_mode
+      type: string
+      mapping:
+        - dps_val: low
+          value: low
+        - dps_val: mid
+          value: medium
+        - dps_val: high
+          value: high
+        - dps_val: auto
+          value: auto
+    # preset only works in cooling mode
+    - id: 8
+      name: preset_mode
+      type: boolean
+      mapping:
+        - dps_val: true
+          value: eco
+        - dps_val: false
+          value: comfort
+    - id: 19
+      name: temperature_unit
+      type: string
+    - id: 105
+      name: swing_mode
+      type: boolean
+      mapping:
+        - dps_val: true
+          value: vertical
+        - dps_val: false
+          value: "off"
+
+secondary_entities:
+  - entity: switch
+    name: Aux Heat

```

Comment:
Aux heat is supported by the climate entity, so this should be moved there.

Reply:
I can test again, however the control for it doesn't show up when under the climate entity